pom.xml

```diff
@@ -383,12 +383,16 @@
                         <goals>
                             <goal>enforce</goal>
                         </goals>
+                        <phase>validate</phase>
                         <configuration>
                             <rules>
                                 <requireJavaVersion>
                                     <!-- Build has not yet been updated for Java 9+ -->
                                     <version>1.8</version>
                                 </requireJavaVersion>
+                                <requireMavenVersion>
+                                    <version>3.6</version>
```

Comment:
This makes you need to update the compiling doc

https://github.com/apache/skywalking/blob/master/docs/en/guides/How-to-build.md#build-from-github

Reply:
At least 3 places should be updated.
1. Prepare git, JDK8, and Maven 3.6+
1. Add document about compiling a single plugin in the `Advanced compile` part.
1. Remove `Setup your Eclipse IDE` section, it is out of the update.

Reply:
Done, Please help to check (my poor english)